### DISCUSSION TITLE
count doesn´t match

### DISCUSSION THREAD

**Discussion by karelcarda** [tchiotludo/akhq#1577] [Q&A] [answered]
in version 0.24.0 (23 is OK)  
the count doesnt match total count of msg :( 
![image](https://github.com/tchiotludo/akhq/assets/120556117/1bdc6a91-0c63-4111-a5ee-2918a97d2959)
VS
Processed a total of 374 messages

**Comment by tchiotludo** [comment, marked as answer]
The count of message is reported by Kafka API, it's a best effort number and we don't have any power on that values
